wled00/data/settings_leds.htm

```diff
@@ -224,6 +225,41 @@
 			let sLC = 0, sPC = 0, sDI = 0, maxLC = 0;
 			const ablEN = d.Sf.ABL.checked;
 			maxB = oMaxB; // TODO make sure we start with all possible buses
+			let setPinConfig = (n,t) => {
+				let p0d = "GPIO:";
+				let p1d = "";
+				let off = "Off Refresh";
+				switch (gT(t).t.charAt(0)) {
+					case '2': // 2 pin digital
+						p1d = "Clock "+p0d;
+						// fallthrough
+					case 'D': // digital
+						p0d = "Data "+p0d;
+						break;
+					case 'A': // PWM analog
+						if (gT(t).t.length > 1) p0d = "GPIOs:";
+						off = "Dithering";
+						break;
+					case 'N': // network
+						p0d = "IP address:";
+						break;
+					case 'V': // virtual/non-GPIO based
+						p0d = "Config:"
+						break;
+				}
```

Comment:
This is definitely a good step in the right direction, which I hope to improve in the next PR where this is returned by the Bus type rather than hard coded in the HTML - e.g hub75 having "chain length", panel size etc